### PR TITLE
[i18n] Add resources for Russian language

### PR DESCRIPTION
**Pull request by Sveer**
@RocketChat/android
Add resources for Russian language (app & widget):
screenshot_2017-12-11-00-42-52-670_chat rocket android
screenshot_2017-12-11-00-45-32-166_chat rocket android

### CODE REVIEW

**Comment by rafaelks** [comment]
LGTM! Thank you @Sveer!

**Review by rafaelks — APPROVED** [review]

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    <string name="user_status_online">Онлайн</string>
+    <string name="user_status_away">Отошел</string>
+    <string name="user_status_busy">Занят</string>
+    <string name="user_status_invisible">Невидимый</string>
+    <string name="fragment_sidebar_main_logout_title">Выход</string>
+    <string name="dialog_add_channel_name">Название</string>
+    <string name="dialog_add_channel_private">Закрытый</string>
+    <string name="dialog_add_channel_read_only">Только для чтения</string>
+
+    <string name="fragment_room_list_pinned_message_title">Закрепленные сообщения (%с)</string>
```
Comment (leonardoaramaki) [inline app/src/main/res/values-ru/strings.xml]: @Sveer Thanks for contributing!
Does this formatter qualifier `(%с)` works for you? Here it crashes the app.

**Review by leonardoaramaki — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (Sveer) [reply app/src/main/res/values-ru/strings.xml]: Ou.. Sorry :( Will fix.

**Review by leonardoaramaki — APPROVED** [review]
Thanks @Sveer!

**Pull request merged**